cluster/gce/configure-vm.sh

```diff
@@ -968,7 +968,7 @@ function salt-node-role() {
 grains:
   roles:
     - kubernetes-pool
-  cbr-cidr: 10.123.45.0/30
+  cbr-cidr: 10.123.45.0/29
```

Comment:
Also please comb through docs to ensure that we're not infringing on a subnet that we previously didn't infringe on. see: https://cloud.google.com/compute/docs/vm-ip-addresses and https://cloud.google.com/compute/docs/networking#subnet_network